build/deploy/grafana_dashboards/crdb-runtime-grafana.json

```diff
@@ -81,7 +81,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "count(up{app=\"cockroachdb\",cluster_name=\"$cluster_name\",pod_name=~\"$node\"})",
+          "expr": "count(up{app=\"cockroachdb\",k8s_cluster=\"$k8s_cluster\",pod_name=~\"$node\"})",
```

Comment:
in this case can you change it to k8s_cluster=~\"^$k8s_cluster.*$\"
where ever applicable this way we will get to use multiselect, and aggregate multiple clusters together

Reply:
done